tests/test_segments.py

```diff
@@ -881,10 +881,10 @@ class TestBat(TestCommon):
 	def test_battery(self):
 		pl = Pl()
 
-		def _get_capacity(pl):
-			return 86
+		def _get_battery_status(pl):
+			return 86, False
```

Comment:
There should be more tests:

1. Copy of the current one, but with `True` as the battery status second result.
2. Explicit `online` and `offline` keywords and `True`.
3. Explicit `online` and `offline` keywords and `False`.